docs/Using-Pantheon/Truffle.md

```diff
@@ -0,0 +1,59 @@
+description: Using Pantheon with Truffle
+<!--- END of page meta data -->
+
+# Using Pantheon with Truffle 
+
+Developing for Pantheon using Truffle is the same as using Truffle to develop for the public Ethereum networks. 
+Truffle supports Pantheon with the only difference being Pantheon does not implement private key management. 
+To use Pantheon with Truffle, you must configure the Truffle wallet.
+
+## Install Truffle Wallet 
+
+To install the [Truffle wallet](https://github.com/trufflesuite/truffle-hdwallet-provider):
```

Comment:
This seems to be deprecated now. We might need to update it soon. I believe the instructions will be the same. I'm not sure on what we need to update though.

Reply:
yeah instructions are the same, maybe just remove the github link

Reply:
Done